Purchases/Networking/RCHTTPClient.m

```diff
@@ -10,13 +10,16 @@
 #import "RCLogUtils.h"
 #import "RCHTTPStatusCodes.h"
 #import "RCSystemInfo.h"
+#import "RCHTTPRequest.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface RCHTTPClient ()
 
 @property (nonatomic) NSURLSession *session;
-@property (nonatomic, weak) RCSystemInfo *systemInfo;
+@property (nonatomic) RCSystemInfo *systemInfo;
```

Comment:
this is now important - the `RCHTTPClient` could live longer than other objects, and it needs systemInfo to be alive when starting more requests. so it needs to hold it strongly.